playbooks/roles/integreatly/files/rhmi_post_install_housekeeping_survey.json

```diff
@@ -0,0 +1,39 @@
+{
+    "spec": [
+      {
+        "question_description": "The name of the cluster ie, console.<CLUSTER NAME>.openshift.com",
```

Comment:
Would it not be better for this value to just be the cluster name? i.e. rhmi-dev1. Having to type in `console.<CLUSTER NAME>.openshift.com` differs from all other surveys where you just need to pass in the cluster name

Reply:
On second look, maybe the value is just the cluster name and the description might be misleading me here by referencing `console.<CLUSTER NAME>.openshift.com `

Reply:
I thought the same yesterday. Its only cluster_name that is needed.